### PR TITLE
Fixes #15476: rudder-pkg does not install properly

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15476

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR rebased

File: relay/sources/Makefile [hunk 1]
```diff
@@ -43,6 +44,9 @@ relay-api/virtualenv/virtualenv.py: relay-api/virtualenv.tgz
 	cd relay-api && tar -xzf ./virtualenv.tgz
 	cd relay-api && mv ./virtualenv-$(VIRTUALENV_RELEASE)/ virtualenv/
 
+rudder-pkg/rudder_plugins_key.pub:
+	$(GET) rudder-pkg/rudder_plugins_key.pub https://repository.rudder.io/apt/rudder_apt_key.pub $(RUDDER_PLUGINS_KEY_SHA256 )
```
Comment (amousset) [inline relay/sources/Makefile]: space after SHA256

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**